src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -2742,17 +2742,17 @@ public void RecoverableTree_With(string language)
 
             var document = sol.GetDocument(did);
             var tree = document.GetSyntaxTreeSynchronously(default);
-            var recoverableTree = Assert.IsAssignableFrom<IRecoverableSyntaxTree>(tree);
+            //var recoverableTree = Assert.IsAssignableFrom<IRecoverableSyntaxTree>(tree);
```

Comment:
This test should just be deleted right? Also should anything calling CreateWorkspaceWithRecoverableSyntaxTreesAndWeakCompilations go away too?

Reply:
will do in followup!

Reply:
done in followup.